src/main/java/com/spotify/reaper/ReaperApplicationConfiguration.java

```diff
@@ -33,10 +36,12 @@
 
   @JsonProperty
   @NotNull
-  private String repairParallelism;
+  private RepairParallelism repairParallelism;
 
   @JsonProperty
   @NotNull
+  @Min(0)
```

Comment:
Greater than 0 should be enforced. 0 will cause division by zero. Any straightforward way to do this?